### DISCUSSION TITLE
package sleap-1.2.8-py37_1 requires tensorflow 2.6.3.*, but none of the providers can be installed

### DISCUSSION THREAD

**Discussion by ryanschwark** [talmolab/sleap#1506] [Help!]
Hi all,

I'm trying to install SLEAP on my Windows 11 PC but have been getting an error. I've successfully installed SLEAP on this PC before, but had to uninstall, and I'm trying to re-install now. When I use the quick install command (conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap) I get this error:

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package sleap-1.2.8-py37_1 requires tensorflow 2.6.3.*, but none of the providers can be installed

Could not solve for environment specs
The following packages are incompatible
└─ sleap   is uninstallable because there are no viable options
   ├─ sleap [1.2.0|1.2.1|...|1.3.0] would require
   │  └─ tensorflow 2.6.3.* , which conflicts with any installable versions previously reported;
   ├─ sleap [1.3.1|1.3.2|1.3.3] would require
   │  └─ tensorflow >=2.6.3,<2.11 , which conflicts with any installable versions previously reported;
   ├─ sleap [1.1.0|1.1.1|1.1.3|1.1.4|1.1.5] would require
   │  └─ pyside2 5.14.1.* , which conflicts with any installable versions previously reported;
   └─ sleap 1.0.9 would require
      └─ scipy <=1.4.1 , which conflicts with any installable versions previously reported.

What should I do to fix this error? Is this an issue with Tensorflow being the incorrect version? Thanks!

**Comment by talmo** [comment]
Hi @ryanschwark,

Do you get the same error if you run this command?

```
conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3
```

Thanks!

Talmo

**Reply by ryanschwark** [reply]
Hi Talmo,

Yep, if I run conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3, I get this error:

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package sleap-1.3.3-py37_0 requires tensorflow >=2.6.3,<2.11, but none of the providers can be installed

Could not solve for environment specs
The following package could not be installed
└─ sleap 1.3.3**  is not installable because it requires
   └─ tensorflow >=2.6.3,<2.11 , which conflicts with any installable versions previously reported.

**Comment by roomrys** [comment]
Hi @ryanschwark,

Do you mind also pasting the terminal message for which `tensorflow` it is expecting? Those versions for `tensorflow` look correct, but I am wondering why mamba says it is uninstallable. 

Another thing to try would be to clean the cache of packages with
```bash
mamba clean -a
```
as some of the packages that were installed in your old version of SLEAP may take priority over newer versions available for download.

Thanks,
Liezl
